### PR TITLE
Decay benchmarks in test suite

### PR DESCRIPTION
**Pull request by scopatz**
This adds the decay benchmarks as a tests in the test suite.  The original benchmark notebook can be seen here [1]. Unlike the original benchmark, this does not rely on pandas.

1. http://nbviewer.ipython.org/github/pyne/sandbox/blob/master/origen-cmp.ipynb

### CODE REVIEW

**Comment by scopatz** [comment]
relaunching

**Comment by scopatz** [comment]
OMG it works!

File: tests/test_decay.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    * the material key, matkey
+    * the parent nuclide, nuc
+    * the decay time, t
+    * the maximum relative error over all nuclides, relerr
+    * the nuclide with the maximum relative error, child
+    * The mass of the child, childmass
+    * the child's mass weighted relative error, weightederr
+
+    Note that this also filters by when Origen itself has messed up by 
+    gaining or lossing too much mass (1%). Furthermore, the maximum relative
```
Comment (crbates) [inline tests/test_decay.py]: lossing -> losing

**Comment by crbates** [comment]
one minor spelling fix otherwise this is good to go.

**Comment by scopatz** [comment]
Typo fixed.

**Comment by crbates** [comment]
batlab is at a standstill, I will merge when it's done.

**Comment by scopatz** [comment]
Yeah, this is weird.

**Comment by scopatz** [comment]
Looks like this passes!

**Comment by crbates** [comment]
thanks @scopatz!

**Comment by scopatz** [comment]
Woohoo!

**Pull request merged**